examples/test_case_data/sandbox/kylin.properties

```diff
@@ -183,7 +183,8 @@ kylin.web.default-time-filter=0
 # kylin query metrics percentiles intervals default=60, 300, 3600
 kylin.server.query-metrics-percentiles-intervals=60, 360, 3600
 
-# Env DEV|QA|PROD
+# Env DEV|QA|PROD\LOCAL\UT
```

Comment:
split by | ？

Reply:
Sure, it's not so obvious in my IDE between '|' and '\' .